app/ui/src/main/res/values/strings.xml

```diff
@@ -83,7 +88,7 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="delete_messages_label">Delete Emails</string>
     <string name="delete_messages_desc">Allows this application to delete your Emails.</string>
 
-    <string name="about_title_fmt">About <xliff:g id="app_name">%s</xliff:g></string>
```

Comment:
Is there a reason `%s` was removed? I guess some people may want to release a customized version of K-9 mail with a different name that is easy to adjust.

Reply:
A few other strings appear to use "K-9 Mail" instead of string formatting
I thought I'd try to fix all of those in one go later (eg using [xml entities](https://stackoverflow.com/a/39870268))
Would it be preferable to keep them as formatted strings for now?

Reply:
If you're planning to address it later I think it's OK to use "K-9 Mail" now, at least this will be consistent :)